app/(protected)/admin/accounting/page.tsx

```
@@ -46,40 +46,40 @@ export default async function PaymentsPage() {
-          value={`$${totalRevenue.toFixed(2)}`}
+          title="Revenu Total"
+          value={`${totalRevenue.toFixed(2)} €`}
           type="dollarSign"
-          change="+18% from last month"
+          change="+18% par rapport au mois dernier"
         />
         <InfoCard
-          title="Average Payment"
-          value={`$${isNaN(averagePayment) ? '0.00' : averagePayment.toFixed(2)}`}
+          title="Paiement Moyen"
+          value={`${isNaN(averagePayment) ? '0,00' : averagePayment.toFixed(2)} €`}
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)